browser/backend/chalice/Makefile

```diff
@@ -5,7 +5,9 @@ export ACCOUNT_ID=$(shell aws sts get-caller-identity --query Account --output t
 export APP_NAME=browser-api
 export EXPORT_ENV_VARS_TO_LAMBDA=APP_NAME DEPLOYMENT_STAGE
 export PYTHONHASHSEED=0
-
+export S3_DEPLOYMENT_FILE=s3://org-dcp-infra-$(ACCOUNT_ID)/chalice/envs/$(DEPLOYMENT_STAGE)/deployed.json
+export DEPLOYED_PATH= .chalice/deployed
+export DEPLOYED = $(DEPLOYED_PATH)/$(DEPLOYMENT_STAGE).json
```

Comment:
Can these be more explicit/consistent, for example: `S3_DEPLOYED_FILE`, `LOCAL_DEPLOYED_PATH` and `LOCAL_DEPLOYED_FILE`?

Reply:
Also consistent spacing please!